### DISCUSSION TITLE
Description of the last graph

### DISCUSSION THREAD

**Discussion by egor-ryashin** [dirkschreib/SilentStudio#10] [Q&A] [answered]
I wonder how to read this graph? 
![image](https://github.com/dirkschreib/SilentStudio/assets/14215045/6518d553-632f-4b89-9f46-8175e5018856)

**Comment by dirkschreib** [comment, marked as answer]
This looks similar to my graph:
<img width="310" alt="image" src="https://github.com/dirkschreib/SilentStudio/assets/39700157/e180d1b8-5b72-444b-aef3-07efa0f872f8">
But your fan is still off (0 rpm) and the temperature is nearly 70°C (69,8°C). I assume you have shut if off manually or are not using a M1 Mac Studio.
How to read the green and blue line? The blue line shows the rpm values when the temperature is rising. It should switch on the fans at around 65°C. The green line shows the rpm values when the temperature is falling. I.e. it should switch off the fan at ~45°C.
